RevenueCatUI/Data/PaywallData+Validation.swift

```diff
@@ -45,24 +46,26 @@ extension Offering {
 extension Offering {
 
     /// - Returns: a validated paywall suitable to be displayed, and any associated error.
-    func validatedPaywall(locale: Locale = .current) -> ValidationResult {
+    func validatedPaywall(locale: Locale) -> ValidationResult {
         if let paywall = self.paywall {
             switch paywall.validate() {
             case let .success(template):
-                return (paywall, template, nil)
+                return (paywall, paywall.locale ?? locale, template, nil)
```

Comment:
`paywall.locale` SHOULD always exist at this point but needed to do something like this to make it happy 🤷‍♂️